style.css

```diff
@@ -351,3 +352,89 @@ h2 {
   align-items: flex-start;
   gap: 15px;
 }
+
+.contact-form {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  background-image: url(assets/images/background2.png);
```

Comment:
- [ ] Kindly add the background image like the one used in the template you chose, please use the correct path to the image (background2.png doesn't exist) and you have a background image from another template in you assets folder 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/199596587-38761f71-568e-4cd6-8a60-02e1b9780b4a.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/199596621-b260eed7-904b-4b5c-86c0-678eb9dad868.png)